src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import "./App.css";
+import logo from "./logo.png";
+import {BrowserRouter, Routes, Route} from "react-router-dom";
+import Home from "./frontend/pages/home";
+import Appp from "./frontend/pages/product";
+import Cart from "./frontend/pages/cart";
+import Signin from "./frontend/pages/signin";
+import Signup from "./frontend/pages/signup";
+function App() {
```

Comment:
As you have used the barrel export in pages folder, you no longer need to give full path of components. You can only give `./frontend/pages`